__tests__/html/__jest__/setupRunHTMLTest.js

```diff
@@ -21,7 +21,7 @@ global.runHTMLTest = async (
 
   const preferences = new logging.Preferences();
 
-  preferences.setLevel(logging.Type.BROWSER, logging.Level.WARNING);
+  preferences.setLevel(logging.Type.BROWSER, logging.Level.SEVERE);
```

Comment:
why are we changing the logging level?

Reply:
The example code I was looking at changed the level, but I don't think it changes behavior for what I'm testing. I'm going to revert back to WARNING.